core/src/main/scala/org/elasticmq/actor/QueueManagerActor.scala

```diff
@@ -5,13 +5,14 @@ import org.elasticmq.actor.queue.QueueActor
 import org.elasticmq.actor.reply._
 import org.elasticmq.msg.{CreateQueue, DeleteQueue, _}
 import org.elasticmq.util.{Logging, NowProvider}
-import org.elasticmq.{QueueAlreadyExists, QueueData}
+import org.elasticmq._
 
 import scala.reflect._
 
-class QueueManagerActor(nowProvider: NowProvider) extends ReplyingActor with Logging {
+class QueueManagerActor(nowProvider: NowProvider, sqsLimit: SQSLimits) extends ReplyingActor with Logging {
```

Comment:
Maybe let's just call them: `Limits`. `SqsLimits` could then be the instantiation of `Limits` with values characteristic to `Sqs`.

Reply:
Changed them to `Limits` but I have not changed the structure of it yet. I propose to stay with how it is done right now and if someone will have another use case, then it could be remodelled according to the needs.